docs/pages/new-actions.rst

```diff
@@ -20,8 +20,8 @@ actuation_spec contains any parameters needed by that control function.
 See ``habitat_sim/agent/controls/default_controls.py`` for more example
 controls.
 
-Controls are registered using the `habitat_sim.agent.register_move_fn()`
-function. This function takes the functor to register and, optionally, the name
+Controls are registered using `habitat_sim.registry.register_move_fn`.
```

Comment:
Oh, I was always appending the `()` to function references (to distinguish them from variables/properties/modules)... and I did that for the soon-to-be-PR'd habitat-api docs rework as well. Please let me know if I should stop doing that :)

Reply:
I like that!  I didn't realize that's why you had done that so it didn't register that I should keep it.